packages/react/src/components/ExpandCollapsePanel/ExpandCollapsePanel.tsx

```diff
@@ -198,16 +199,15 @@ export default class ExpandCollapsePanel extends React.Component<
             onClick: this.handleToggle
           })}
         <div
-          {...other}
           className={classnames(
-            'ExpandCollapse__panel',
+            className ? className : 'ExpandCollapse__panel',
```

Comment:
This should _always_ have the `ExpandCollapse__panel` className, otherwise we're introducing a breaking change.

Reply:
Would this be breaking if it is always providing it as the default, but allowing for a different class name to be passed in, if the developer wants to? That's more or less what is trying to be accomplished here (among other areas with similar patterns).

Reply:
Additional class names should be appended onto the existing one.